### ISSUE TITLE
found error when use demo code.

### ISSUE THREAD

**Issue by LeToNode** [konsumer/rawproto#8]
Error [ERR_REQUIRE_ESM]: require() of ES Module /{filepath}/node_modules/rawproto/dist/rawproto.modern.js from /{filepath}/example.js not supported.
Instead change the require of rawproto.modern.js in /{filepath}/example.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/{filepath}/example.js:2:29) {
  code: 'ERR_REQUIRE_ESM'
}

node : 16.12.0
npm: 8.19.4

**Comment by LeToNode** [comment]
example.js code 
```
const fs = require('fs')
const {getData ,getProto} = require('rawproto')

var buffer = fs.readFileSync('/Users/abc/Downloads/discover.proto')

// get info about binary protobuf message
console.log( rawproto.getData(buffer) )

// print proto guessed for this data
console.log( rawproto.getProto(buffer) )



```

**Comment by konsumer** [comment]
The error says that you are requiring an ES6 import. I think maybe I need to modify the package.json for CJS. In the meantime, you can add `"type": "module"` to your package.json and use `import` or `require('rawproto/dist/rawproto.cjs')`

**Comment by konsumer** [comment]
should be fixed on `0.7.13`.
